Makefile

```diff
@@ -112,10 +114,11 @@ run-test: build
 	PID=$$!; \
 	echo "skiperator PID: $$PID"; \
 	echo "Log redirected to file: $$LOG_FILE"; \
-	if [ -z "$(TEST_DIR)" ]; then \
-		$(MAKE) test; \
-	else \
-		$(MAKE) test-single dir=$(TEST_DIR); \
-	fi; \
-	echo "Stopping skiperator (PID $$PID)..."; \
-	kill $$PID
```

Comment:
We still need to kill skiperator manually

Reply:
Seems like its catching the wrong PID, at least for me. Ill try to fix it

Reply:
Added it back and tested fine now. Think something with me was just not working properly.